web/src/feature/Status.tsx

```diff
@@ -0,0 +1,54 @@
+import { Banner as EdsBanner, Icon, Progress } from '@equinor/eds-core-react'
+import {
+  error_outlined,
+  info_circle,
+  warning_outlined,
+} from '@equinor/eds-icons'
+import { TCalcStatus, TResults } from '../types'
+
+export const Status = (props: {
+  calcStatus: TCalcStatus
+  result: TResults | undefined
+}) => {
+  switch (props.calcStatus) {
+    case 'calculating':
+      return <Progress.Linear />
+    case 'done':
+      if (props.result && 'Mercury' in props.result.phaseValues) {
+        return (
+          <EdsBanner>
+            <EdsBanner.Icon variant="warning">
+              <Icon data={warning_outlined} />
+            </EdsBanner.Icon>
+            <EdsBanner.Message>
+              Liquid mercury has been formed
+            </EdsBanner.Message>
+          </EdsBanner>
+        )
+      } else {
+        return <></>
+      }
+    case 'failure':
+      return (
+        <EdsBanner>
+          <EdsBanner.Icon variant="warning">
+            <Icon data={error_outlined} />
+          </EdsBanner.Icon>
+          <EdsBanner.Message>
+            An unexpected error has occurred. Please inform us if it persists
```

Comment:
```suggestion
            An unexpected error has occurred. Please inform the system responsible if the problem persists
```

Reply:
I agree, that was better. Already merged it though, but may fix it later